Makefile

```diff
@@ -124,24 +123,27 @@ cluster-clean: $(KUBECTL)
 	$(KUBECTL) delete --ignore-not-found -f deploy/
 	$(KUBECTL) delete --ignore-not-found -f deploy/crds/nmstate_v1alpha1_nodenetworkstate_crd.yaml
 	$(KUBECTL) delete --ignore-not-found -f deploy/crds/nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml
-	if [[ "$$KUBEVIRT_PROVIDER" =~ ^os-.*$$ ]]; then \
+	if [[ ! "$$KUBEVIRT_PROVIDER" =~ ^k8s-.*$$ ]]; then \
```

Comment:
Negative checks are evil, please stay checking for okd-. The fact that something is not kubernetes does not mean it is openshift ;)